<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FORM PEMBAYARAN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="iDALATLabel">
        <property name="text">
         <string>NO NOTA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtNota"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="jENISALATLabel">
        <property name="text">
         <string>DENDA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtDenda"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="tIPEALATLabel">
        <property name="text">
         <string>TOTAL_BIAYA</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtBiaya"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mERKLabel">
        <property name="text">
         <string>JUMLAH BAYAR</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtJmlBayar"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="wARNALabel">
        <property name="text">
         <string>KEMBALIAN</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtKembalian"/>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateTgl">
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tANGGALLabel">
        <property name="text">
         <string>TANGGAL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbPeminjaman"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nOPEMINJAMANLabel">
        <property name="text">
         <string>NO PEMINJAMAN</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cmbPembayaran"/>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="cmbPengembalian"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="sTATUSPEMBAYARANLabel">
        <property name="text">
         <string>STATUS PEMBAYARAN</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="sTATUSPENGEMBALIANLabel">
        <property name="text">
         <string>STATUS PENGEMBALIAN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnTambah">
         <property name="text">
          <string>TAMBAH</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnUpdate">
         <property name="text">
          <string>UPDATE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="btnHapus">
         <property name="text">
          <string>HAPUS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="viewData"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
